src/android/PurchasesPlugin.java

```diff
@@ -27,10 +27,14 @@
 
 public class PurchasesPlugin extends AnnotatedCordovaPlugin {
 
+    public static final String PLATFORM_NAME = "cordova";
+    public static final String PLUGIN_VERSION = "1.2.0-SNAPSHOT";
```

Comment:
Can't use BuildConfig like in React Native since this file gets added by cordova as a separate file not as a module like in React Native